doc/source/extended_documentation/categorical/build_a_decision_tree.rst

```diff
@@ -74,9 +74,12 @@ accessed with this key contains the essentials that make the node function.
     must be exceeded or not exceeded (see threshold_condition) for the node to
     progress to the succeed target. Two values required if condition_combination
     is being used.
-  - **threshold_condition** (str): Defines the inequality test to be applied to
+  - **threshold_condition** (str or list(str)): Defines the inequality test to be applied to
```

Comment:
I think we should have a unit test that demonstrates this new behaviour explicitly, rather than via the `if_masked` functionality.